### ISSUE TITLE
[Support]: 0.11.0-Beta2 No Webinterface 

### ISSUE THREAD

**Issue by AlexanderSch90** [blakeblackshear/frigate#3248]
### Describe the problem you are having

I updated to the latest beta yesterday. After adjusting the ffmpg-config for my Reolink cameras, frigate started normally. The detection via mqqt also works so far. However, I have no access to the web interface (logo present in the upper right and rotating circle in the middle of the screen). The page is not loaded correctly. After some time, errors appear in the Docker log. Attached is the log. Anybody have an idea?
I am accessing Frigate from a different subnet (camera system on subnet 10.4.90.xx and I am in 10.4.4.xx. Think this should not be a problem though. It wasn't with all other versions either. 

### Version

0.11.0-d2c3cdc

### Frigate config file

```yaml
detectors:
  coral:
    type: edgetpu
    device: usb
mqtt:
  host: 10.4.4.15
  port: 1883
  # Optional: topic prefix (default: shown below)
  # WARNING: must be unique if you are running multiple instances
  topic_prefix: frigate
  # Optional: client id (default: shown below)
  # WARNING: must be unique if you are running multiple instances
  client_id: frigate
  # Optional: user
  user: mqtt_user
  # Optional: password
  # NOTE: Environment variables that begin with 'FRIGATE_' may be referenced in {}.
  #       eg. password: '{FRIGATE_MQTT_PASSWORD}'
  password: password
  # Optional: interval in seconds for publishing stats (default: shown below)
  stats_interval: 60


ffmpeg:
  hwaccel_args: -hwaccel_output_format qsv -c:v h264_qsv
  input_args: -avoid_negative_ts make_zero -flags low_delay -fflags discardcorrupt -strict experimental -rw_timeout 5000000 -use_wallclock_as_timestamps 1 -f live_flv
  output_args:
    record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c copy # -an
    
objects:
  # Optional: list of objects to track from labelmap.txt (default: shown below)
  track:
    - person
    - car
    - dog
  # Optional: filters to reduce false positives for specific object types
  filters:
    person:
      # Optional: minimum width*height of the bounding box for the detected object (default: 0)
      min_area: 0
      # Optional: maximum width*height of the bounding box for the detected object (default: 24000000)
      max_area: 100000
      # Optional: minimum score for the object to initiate tracking (default: shown below)
      min_score: 0.5
      # Optional: minimum decimal percentage for tracked object's computed score to be considered a true positive (default: shown below)
      threshold: 0.7
    car:
      # Optional: minimum width*height of the bounding box for the detected object (default: 0)
      min_area: 0
      # Optional: maximum width*height of the bounding box for the detected object (default: 24000000)
      max_area: 24000000
      # Optional: minimum score for the object to initiate tracking (default: shown below)
      min_score: 0.5
      # Optional: minimum decimal percentage for tracked object's computed score to be considered a true positive (default: shown below)
      threshold: 0.7
      
record:
  enabled: True
  retain:
    days: 0
    mode: motion
  events:
    max_seconds: 300
    pre_capture: 5
    post_capture: 5
    objects:
      - person
      - car
      - dog
    required_zones: []
    retain:
      default: 20
      mode: active_objects
      objects:
        dog: 2
  
cameras:
  Carport_Einfahrt:
    ffmpeg:
      inputs:
        - path: 'rtmp://10.4.90.21/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - detect
            - rtmp
        - path: 'rtmp://10.4.90.21/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - record
    snapshots:
      enabled: True
    motion:
      mask: 
        - 0,159,283,127,640,59,640,0,0,0
    objects:
      filters:
        car:
          mask: 
            - 0,159,283,127,640,59,640,0,0,0
        person:
          mask: 
            - 0,159,283,127,640,59,640,0,0,0
        dog:
          mask: 
    detect:
      width: 640
      height: 480
      fps: 7
      max_disappeared: 70

  
  Carport_zum_Haus:
    ffmpeg:
      inputs:
        - path: 'rtmp://10.4.90.20/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - detect
            - rtmp
        - path: 'rtmp://10.4.90.20/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - record
    snapshots:
      enabled: True
    motion:
      mask: 
        - 0,480,125,480,243,230,0,238
    objects:
      filters:
        car:
          mask: 
            - 0,480,125,480,243,230,0,238
            - 640,480,640,0,358,87,377,185
        person:
          mask: 
            - 0,480,125,480,243,230,0,238
        dog:
          mask: 
            - 0,480,125,480,243,230,0,238
    detect:
      width: 640
      height: 480
      fps: 7
      max_disappeared: 70
    #detect: alte Kamera
      #width: 640
      #height: 352
      #fps: 6
      #max_disappeared: 60
 
  Hauseingang:
    ffmpeg:
      inputs:
        - path: 'rtmp://10.4.90.11/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - detect
            - rtmp
        - path: 'rtmp://10.4.90.11/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - record
    snapshots:
      enabled: True
    motion:
      mask: 
        - 640,480,640,0,373,0,405,480
    objects:
      filters:
        person:
          mask: 
            - 640,480,640,0,373,0,405,480
        car:
          mask: 
            - 640,480,640,0,373,0,405,480
        dog:
          mask: 
            - 640,480,640,0,373,0,405,480
    detect:
      width: 640
      height: 480
      fps: 7
      max_disappeared: 70
  Hof:
    ffmpeg:
      inputs:
        - path: 'rtmp://10.4.90.30/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - detect
            - rtmp
        - path: 'rtmp://10.4.90.30/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - record
    snapshots:
      enabled: True
    detect:
      width: 640
      height: 480
      fps: 7
      max_disappeared: 70
  Hof_Pixx:
    ffmpeg:
      inputs:
        - path: 'rtmp://10.4.90.34/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - detect
            - rtmp
        - path: 'rtmp://10.4.90.34/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - record
    snapshots:
      enabled: True
    detect:
      width: 640
      height: 480
      fps: 7
      max_disappeared: 70
  Hof_Werkstatt_Mistfall:
    ffmpeg:
      inputs:
        - path: 'rtmp://10.4.90.33/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - detect
            - rtmp
        - path: 'rtmp://10.4.90.33/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - record
    snapshots:
      enabled: True
    detect:
      width: 640
      height: 480
      fps: 7
      max_disappeared: 70
  Hof_zum_Funkturm:
    ffmpeg:
      inputs:
        - path: 'rtmp://10.4.90.32/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - detect
            - rtmp
        - path: 'rtmp://10.4.90.32/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - record
    snapshots:
      enabled: True
    motion:
      mask:
        - 0,249,135,210,130,0,0,0
        - 375,190,640,202,640,0,373,0
        - 640,71,640,0,0,0,0,58
    objects:
      filters:
        car:
          mask: 
            - 0,249,135,210,130,0,0,0
            - 375,190,640,202,640,0,373,0
            - 640,71,640,0,0,0,0,58
        person:
          mask: 
            - 0,249,135,210,130,0,0,0
            - 375,190,640,202,640,0,373,0
            - 640,71,640,0,0,0,0,58
        dog:
          mask: 
            - 0,249,135,210,130,0,0,0
            - 375,190,640,202,640,0,373,0
            - 640,71,640,0,0,0,0,58
    detect:
      width: 640
      height: 480
      fps: 7
      max_disappeared: 70
  Hof_zum_Wintergarten:
    ffmpeg:
      inputs:
        - path: 'rtmp://10.4.90.31/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - detect
            - rtmp
        - path: 'rtmp://10.4.90.31/bcs/channel0_main.bcs?channel=0&stream=0&user=admin&password=PASSWORD'
          roles:
            - record
    snapshots:
      enabled: True
    motion:
      mask:
        - 105,189,288,204,640,390,640,0,156,0,149,113
    objects:
      filters:
        car:
          mask: 
            - 105,189,288,204,640,390,640,0,156,0,149,113
        person:
          mask: 
            - 105,189,288,204,640,390,640,0,156,0,149,113
        dog:
          mask: 
            - 105,189,288,204,640,390,640,0,156,0,149,113
    detect:
      width: 640
      height: 480
      fps: 7
      max_disappeared: 70
```

### Relevant log output

```shell
[2022-05-24 14:57:52] frigate.app                    INFO    : Starting Frigate (0.11.0-d2c3cdc)
Starting migrations
[2022-05-24 14:57:52] peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
[2022-05-24 14:57:52] peewee_migrate                 INFO    : There is nothing to migrate
[2022-05-24 14:57:52] detector.coral                 INFO    : Starting detection process: 216
[2022-05-24 14:57:52] frigate.app                    INFO    : Output process started: 218
[2022-05-24 14:57:52] frigate.app                    INFO    : Camera processor started for Carport_Einfahrt: 225
[2022-05-24 14:57:52] frigate.edgetpu                INFO    : Attempting to load TPU as usb
[2022-05-24 14:57:52] frigate.app                    INFO    : Camera processor started for Carport_zum_Haus: 227
[2022-05-24 14:57:55] frigate.edgetpu                INFO    : TPU found
[2022-05-24 14:57:52] frigate.app                    INFO    : Camera processor started for Hauseingang: 228
[2022-05-24 14:57:52] frigate.app                    INFO    : Camera processor started for Hof: 230
[2022-05-24 14:57:52] frigate.app                    INFO    : Camera processor started for Hof_Pixx: 231
[2022-05-24 14:57:52] frigate.app                    INFO    : Camera processor started for Hof_Werkstatt_Mistfall: 232
[2022-05-24 14:57:52] frigate.app                    INFO    : Camera processor started for Hof_zum_Funkturm: 233
[2022-05-24 14:57:52] ws4py                          INFO    : Using epoll
[2022-05-24 14:57:52] frigate.app                    INFO    : Camera processor started for Hof_zum_Wintergarten: 235
[2022-05-24 14:57:52] frigate.app                    INFO    : Capture process started for Carport_Einfahrt: 236
[2022-05-24 14:57:52] frigate.app                    INFO    : Capture process started for Carport_zum_Haus: 237
[2022-05-24 14:57:52] frigate.app                    INFO    : Capture process started for Hauseingang: 239
[2022-05-24 14:57:52] frigate.app                    INFO    : Capture process started for Hof: 242
[2022-05-24 14:57:52] frigate.app                    INFO    : Capture process started for Hof_Pixx: 244
[2022-05-24 14:57:52] frigate.app                    INFO    : Capture process started for Hof_Werkstatt_Mistfall: 249
[2022-05-24 14:57:52] frigate.app                    INFO    : Capture process started for Hof_zum_Funkturm: 254
[2022-05-24 14:57:52] frigate.app                    INFO    : Capture process started for Hof_zum_Wintergarten: 255
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/usr/local/lib/python3.9/dist-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/usr/local/lib/python3.9/dist-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
TimeoutError: [Errno 110] Connection timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/usr/local/lib/python3.9/dist-packages/urllib3/connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "/usr/local/lib/python3.9/dist-packages/urllib3/connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "/usr/local/lib/python3.9/dist-packages/urllib3/connection.py", line 358, in connect
    self.sock = conn = self._new_conn()
  File "/usr/local/lib/python3.9/dist-packages/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x7f3d9e7696d0>: Failed to establish a new connection: [Errno 110] Connection timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/usr/local/lib/python3.9/dist-packages/urllib3/connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "/usr/local/lib/python3.9/dist-packages/urllib3/util/retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/blakeblackshear/frigate/releases/latest (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3d9e7696d0>: Failed to establish a new connection: [Errno 110] Connection timed out'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/opt/frigate/frigate/__main__.py", line 16, in <module>
    frigate_app.start()
  File "/opt/frigate/frigate/app.py", line 359, in start
    self.init_stats()
  File "/opt/frigate/frigate/app.py", line 154, in init_stats
    self.stats_tracking = stats_init(self.camera_metrics, self.detectors)
  File "/opt/frigate/frigate/stats.py", line 41, in stats_init
    "latest_frigate_version": get_latest_version(),
  File "/opt/frigate/frigate/stats.py", line 23, in get_latest_version
    request = requests.get(
  File "/usr/local/lib/python3.9/dist-packages/requests/api.py", line 75, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.9/dist-packages/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /repos/blakeblackshear/frigate/releases/latest (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3d9e7696d0>: Failed to establish a new connection: [Errno 110] Connection timed out'))
```

### FFprobe output from your camera

```shell
?
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

reolink 410

**Comment by NickM-27** [comment]
This was fixed in #3246 

The container can't connect to github.com which is what it uses to see if there is an update available.

**Comment by AlexanderSch90** [comment]
your right thanks!
